src/core/xref.js

```diff
@@ -72,6 +87,50 @@ export async function run(conf, elems) {
   addDataCiteToTerms(results, xrefMap, conf);
 }
 
+/**
+ * converts conf.xref to object with url and spec properties
+ */
+function normalizeConfig(conf) {
+  const xrefDefaults = {
+    url: API_URL,
+    profile: "",
+    specs: null,
```

Comment:
I'm not sure if we want both `profile` and `specs`. @marcoscaceres Do we plan to add something more based on profile? If not, I think we can have only `specs: null | string | string[]` and if `typeof specs === "string"`, then treat it as profile and convert it to array of specs based on profile.

Reply:
> @marcoscaceres Do we plan to add something more based on profile?

Yes, I was thinking "WHATWG" profile for instance, which doesn't include any w3c specs. And there are a few others, like Web Perf working group would have w3c + their specs.

Reply:
(we would eventually need to figure out some extensible way of creating profiles... but there might be profiles that are popular enough to always include.... specially for w3c specs)

Reply:
I meant, do we plan to use `profile` for something other than a list of specs?

Reply:
oh, apologies, I see what you mean. No, we don't plan to use it for anything else.

Reply:
OK, I was thinking it sounds more natural for the user to write `xref: { profile: "w3c"}` than `xref: { specs: "w3c"}`. But internally, yes we can have only `xref: { url: , specs: }`.

Reply:
I agree with @Swapnilr1 ... there is also less type swapping. I'd prefer to only have multiple types at the `xref` level.